src/test/java/org/jenkinsci/plugins/workflow/cps/FlowDurabilityTest.java

```diff
@@ -736,12 +737,13 @@ public void testResumeBlocked() throws Exception {
             @Override
             public void evaluate() throws Throwable {
                 Jenkins jenkins = story.j.jenkins;
-                WorkflowRun run = createAndRunSleeperJob(story.j.jenkins, jobName, FlowDurabilityHint.MAX_SURVIVABILITY);
-                run.getParent().setResumeBlocked(true);
```

Comment:
This was happening too late—after the build had started. (That corner case is covered in the next test. This test covers the normal case that the job is permanently marked incapable of resuming.)